File: app/src/main/java/com/wwxiaoqi/wechat_backup/utils/TimeUtils.java
package com.wwxiaoqi.wechat_backup.utils;

import java.text.SimpleDateFormat;
import java.util.Calendar;
import java.util.Locale;

/**
 * 时间工具类
 */
public class TimeUtils {

  /***
   * 获取当前时间日期
   * @return 返回 yyyyMMdd 格式
   */
  public static String getDataTime() {
    SimpleDateFormat format = new SimpleDateFormat("yyyyMMdd", Locale.CHINA);
    return format.format(Calendar.getInstance().getTime());
  }

}

File: app/src/main/java/com/wwxiaoqi/wechat_backup/utils/TimeUtils.kt
package com.wwxiaoqi.wechat_backup.utils

import java.text.SimpleDateFormat
import java.util.*

/**
 * 时间工具类
 */
object TimeUtils {
  /***
   * 获取当前时间日期
   * @return 返回 yyyyMMdd 格式
   */
  val dataTime: String
    get() {
      val format = SimpleDateFormat("yyyyMMdd", Locale.CHINA)
      return format.format(Calendar.getInstance().time)
    }
}